packages/cli/src/commands/styles/utils/publish.main.test.ts

```diff
@@ -112,50 +112,44 @@
 // });
 
 
-describe('placeholder test', ()=> {
-  it('placeholder', ()=> {
-    expect(1).toEqual(1);
-  });
-});
-// describe('getStyles, getStyleNodes and processStyles', () => {
-//   it('should fail', async() => {
-//     const figmaAPI = figmaAPIFactory('x-x-x-x-x-x');
-//     const styles = await figmaAPI.getStyles('a bad url');
-//     expect(styles.nodes).toStrictEqual([ '20008:10969', '20029:15098', '20008:10966' ]);
+// describe('placeholder test', ()=> {
+//   it('placeholder', ()=> {
+//     expect(1).toEqual(1);
 //   });
 // });
+describe('getStyles, getStyleNodes and processStyles', () => {
+  it('should fail', async() => {
+    mockedAxios.get.mockRejectedValue(publishStyles);
+    expect.assertions(1);
+    try {
+      const figmaAPI = figmaAPIFactory('x-x-x-x-x-x');
+      await figmaAPI.getStyles('a bad url');
+    } catch (error: any){
+      expect(error?.message).toBe('Failed to parse figma url');
+    }
+  });
 
-//   it('test geting style should return node list and style list', async ()  => {
-//     mockedAxios.get.mockResolvedValueOnce(exampleStyleData);
-//     const figmaAPI = figmaAPIFactory('x-x-x-x-x-x');
-//     const stylesListObj = await figmaAPI.getStyles('TJzz7ZB6pJvpLhjI5DWG3F');
-//     expect(stylesListObj.nodes).toStrictEqual([ '20008:10969', '20029:15098', '20008:10966' ]);
-//     expect(Object.keys(stylesListObj.styles).length).toBe(3);
-//   });
+  it('test geting style should return node list and style list', async ()  => {
+    mockedAxios.get.mockResolvedValueOnce(publishStyles);
+    const figmaAPI = figmaAPIFactory('x-x-x-x-x-x');
+    const styles = await figmaAPI.getStyles('TJzz7ZB6pJvpLhjI5DWG3F');
```

Comment:
hmm we'll have to mock the call `figmaAPI.getStyles`. I'll have to look into the best way to do it but try this below. 

```
// import { figmaAPIFactory } from './publish.main';
spyOn(figmaAPIFactory, 'processStyles').and.returnValue(mockStyles);
```
mockStyles can be the object we expect the api to return, without the api returning anything